test/index.test.ts

```diff
@@ -44,4 +48,14 @@ describe('getValidators', () => {
         const validators = getValidators('1.3.0', <any>null, <any>null, <any>null, <any>null, <any>null);
         expect(validators && validators.length).toEqual(27);
     });
+
+    it('should return amount of validators for version >= 1.4.0', () => {
+        const validators = getValidators('1.4.0', <any>null, <any>null, <any>null, <any>null, <any>null);
+        expect(validators && validators.length).toEqual(26);
+    });
+
+    it('should return amount of validators for version >= 1.5.0', () => {
+        const validators = getValidators('1.5.0', <any>null, <any>null, <any>null, <any>null, <any>null);
+        expect(validators && validators.length).toEqual(26);
```

Comment:
There is one validator less because in 1.4 we dropped a validator related to a specific option of Adobe AEM.